src/data/sign.lean

```diff
@@ -105,6 +105,54 @@ def fin3_equiv : sign_type ≃* fin 3 :=
   end,
   map_mul' := λ x y, by casesm* _; refl }
 
```

Comment:
Might be nice to wrap these in a section as something like:
```
section case_bashing
```
or something, just as there are so many of these.

Reply:
I've added the section and put each such lemma and its proof on a single line.

Given these lemmas, `simp` can prove all true `<` and `≤` relations between any two of `-1`, `0`, `1` in `sign_type, except for:

```lean
@[simp] lemma neg_one_lt_one : (-1 : sign_type) < 1 := by dec_trivial!
```

Should I add that lemma as well so that `simp` can then prove all such true inequalities?

Reply:
I've now added `neg_one_lt_one`, but proved with `bot_lt_top` in the spirit of the other changes to use `order_bot` or `order_top` lemmas.